### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by HarnoorSethi** [ashton0223/quizlet-sets#1]
I tried to use the sample provided, but it does not work anymore. This many be due to a change in quizlet, but idk

**Comment by HarnoorSethi** [comment]
Actually, the reason is because bs4 keeps getting blocked by quizlet

**Comment by HarnoorSethi** [comment]
<response 403> is common, that probably  is the reason

**Comment by violet-isle** [comment]
yeah, module doesn't work anymore, i feel like it should be removed from pypi or archived from github at least.
